ingestion/create_cadet_databases_source/source.py

```diff
@@ -279,11 +289,22 @@ def _get_databases_with_domains_and_display_tags(
                         logging.debug(f"{database} - has no database level metadata")
 
                     database_metadata_dict["domain"] = fqn[1]
-                    database_metadata_tuple = tuple(database_metadata_dict.items())
+                    database_tags = database_metadata_dict.get("tags", [])
+                    # A copy is needed to avoid changing the original dict, whish is reused
```

Comment:
I'm not following the rationale here, as 2 lines above we do change the original dict. Should we consistently operate on a copy?